Sources/XcodeSupport/XcodeProjectDriver.swift

```diff
@@ -19,8 +19,33 @@ public final class XcodeProjectDriver {
         }
 
         // Ensure targets are part of the project
-        let targets = project.targets.filter { configuration.targets.contains($0.name) }
-        let invalidTargetNames = Set(configuration.targets).subtracting(targets.map { $0.name })
+        var configurationTargets = Set(configuration.targets)
+        var invalidTargetNames: [String] = []
+
+        let targets = project.targets.filter {
+            if configuration.targets.contains($0.name) {
+                configurationTargets.remove($0.name)
+                return true
+            }
+
+            invalidTargetNames.append($0.name)
```

Comment:
In my testing, this causes all targets in my .xcodeproj that are not passed to `periphery` to be considered invalid targets.